library/src/morpheus-master/sass/modules/quicklinks/_base.scss

```diff
@@ -3,6 +3,7 @@
   border-right: 1px solid $tool-border-color;
   @media #{$phone}{
     display: inline-block;
+    margin-top: 3px;
```

Comment:
I think it would be better to flex the parent container and use flex for center alignment rather than adding pixel margins. 

The dividing bar between the Quick Links icon and the account drop-down is shorter than the others.

Something like this would be better:
```
.Mrphs-quickLinksNav__popup {
	display: flex;
	/* margin-top: 3px; */
	height: 100%;
	align-items: center;
}
```

Reply:
I agree with you @fostersdesign , using "display: flex" seems to be better practice.

I made another PR https://github.com/sakaiproject/sakai/pull/5807 to fix this and improve the quick links box.